scripts/_gup.js

```diff
@@ -1,3 +1,13 @@
+/**
+ * Returns the value of a query-string parameter.
+ *
+ * TODO: Why it is called "gup"?
+ *
+ * TODO @ildiria: This method strikes me as completely unnecessary and overly complicated. Moreover it is only used in a single place. Why don't we simply use (https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get#example) in the place where we need it?
```

Comment:
Legacy Tobias.

Reply:
I don't think I've ever even noticed this file :smiley:.